src/masonite/stubs/notification/create_notifications_table.py

```diff
@@ -0,0 +1,17 @@
+from masoniteorm.migrations import Migration
+
+
+class CreateNotificationsTable(Migration):
+    def up(self):
+        """Run the migrations."""
+        with self.schema.create("notifications") as table:
+            table.string("id", 36).primary()
```

Comment:
Usually there are a lot of notifications right? Should we default this to `big_increments`?

Reply:
Yeah maybe, here I was using id as `uuid` but UUID column in ORM was not released at the time I wrote this line.
I propose to keep uuid but to use the real uuid column from ORM. Are you ok ?

Reply:
It's now fixed, I just need to update the database in the sqlite file.

Reply:
I actually used a big_increments, no need to hide the id I guess